fairmq/tools/runSimpleMQStateMachine.h

```diff
@@ -30,6 +32,15 @@ inline int runStateMachine(TMQDevice& device, FairMQProgOptions& cfg)
         return 0;
     }
 
+    if (cfg.Count("version"))
+    {
+        std::cout << "User device version: " << device.GetVersion() << std::endl;
```

Comment:
I think "User ..." is more clear because it prints the version user gives to a device that he derives from FairMQDevice, maybe a better name?. The FairMQDevice version is currently FAIRMQ_INTERFACE_VERSION, which is also printed, but this we can replace with FairRoot version or our own FairMQ semver.